# Optimizes the trajectory and footholds to reach a given goal state
# The optimized values are saved in the server and can be retrieved by another
# service call.

StateLin3d curr_state
Foothold[] curr_stance 
Foothold[] steps